deptrac.config.yaml

```diff
@@ -116,19 +114,19 @@ deptrac:
       - File
     Dependency:
       - Ast
-    Result:
-      - Dependency
+#    Result:
```

Comment:
Why is this commented out?

Reply:
We don't have those Layers configured. I think someone forgot to delete those rulesets previously.